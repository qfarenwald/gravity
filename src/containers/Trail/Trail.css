.Trail {
  align-items: center;
  background-image: linear-gradient(to right, rgba(56,180,73,.5), rgba(56,180,73,.9));
  border-radius: 50px;
  color: #FFFFFF;
  display: flex;
  justify-content: center;
  margin: 15px;
}

.Trail-p {
  letter-spacing: 5px;
  padding: 0 15px 0 15px;
}

.stars {
  width: 20%
}

.name {
  width: 80%
}

.Trail-btn {
  background-color: #38B449;
  border: none;
  border-radius: 0 25px 25px 0;
  font-size: 1rem;
  letter-spacing: 5px;
  padding: 20px;
  height: 100%;
}

.Trail-btn:hover {
  background-color: #33993D;
  color: white;
  cursor: pointer;
}
