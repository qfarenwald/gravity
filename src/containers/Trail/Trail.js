import React from 'react';
import './Trail.css';

export const Trail = () => {
  return (
    <section className="Trail">
      <p>hello</p>
    </section>
  )
}
