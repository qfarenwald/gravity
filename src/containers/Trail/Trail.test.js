import React from 'react';
import ReactDOM from 'react-dom';
import { Trail } from './Trail';
import { shallow } from 'enzyme';

describe('Trail', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<Trail />)

    expect(wrapper).toMatchSnapshot()
  })
})
