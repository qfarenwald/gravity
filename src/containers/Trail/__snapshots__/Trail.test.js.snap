// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Trail should match snapshot 1`] = `
<section
  className="Trail"
  id="trail"
>
  <p
    className="Trail-p stars"
  >
     Stars
  </p>
  <p
    className="Trail-p name"
  />
  <p
    className="Trail-p descent"
  >
     ft Descent
  </p>
  <Link
    to="/shuttle/undefined"
  >
    <button
      className="Trail-btn"
    >
      BOOK
    </button>
  </Link>
</section>
`;
